### PR TITLE
Chore/css fixes

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

Fix for css that had some merge issues.

### CODE REVIEW

File: assets/sass/cds/_components.scss [hunk 1]
```diff
@@ -1,138 +1,159 @@
...
-	max-width: 40rem;
-	margin: 0 auto;
+  max-width: 40rem;
+  margin: 0 auto;
 }
 
 // Adapted from WET's Bootstrap based alerts
 // https://wet-boew.github.io/v4.0-ci/demos/collapsible-alerts/collapsible-alerts-en.html
 .info-alert {
+  background: #d7faff;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:145]: Is this new?

File: assets/sass/cds/_components.scss [hunk 2]
```diff
@@ -146,16 +167,103 @@
...
-	margin-bottom: 2rem;
-	color: white;
-
+  padding: 2.5rem;
+  background-color: $team-purple;
+  margin-bottom: 2rem;
+  color: white;
 }
 .team-listing {
+  list-style: none;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:178]: Is this new?

File: assets/sass/cds/_components.scss [hunk 3]
```diff
@@ -146,16 +167,103 @@
...
+
+    @media (max-width: 1265px) {
+      padding: 1.45rem 1.45rem;
+    }
+
+    &:hover {
+      background: $primary-color;
+    }
+  }
 	list-style: none;
```
Comment (dj2) [inline assets/sass/cds/_components.scss:267]: Is the indenting here off?

File: assets/sass/cds/_components.scss [hunk 4]
```diff
@@ -251,91 +359,204 @@
...
+
+    @include desktop {
+      font-size: 3.5rem;
+    }
+  }
 }
 
 // Blogpost page
 .page-blogpost {
+  h1 {
```
Comment (dj2) [inline assets/sass/cds/_components.scss:444]: New?

File: assets/sass/cds/_components.scss [hunk 5]
```diff
@@ -458,7 +679,66 @@
 }
 
 .meet-the-team-heading {
-	font-size: 45px;
-	margin-bottom: 4rem;
+  font-size: 45px;
+  margin-bottom: 4rem;
 }
 
+.job-postings-container {
```
Comment (dj2) [inline assets/sass/cds/_components.scss:686]: new?

File: assets/sass/cds/_cta.scss [hunk 6]
```diff
@@ -126,5 +126,9 @@
     }
 }
 
-
+.black {
```
Comment (dj2) [inline assets/sass/cds/_cta.scss:129]: new?

File: assets/sass/cds/_sections.scss [hunk 7]
```diff
@@ -411,135 +417,127 @@ span.livefr {
     }
   }
 
-  p, a {
-    font-size: 1.85rem;
-    display: inline-block;
-  }
-
+  h5,
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:420]: Where did the h5 come from? This was removed I believe?

File: assets/sass/cds/_sections.scss [hunk 8]
```diff
@@ -411,135 +417,127 @@ span.livefr {
...
 
-    @include mobile_only {
-      padding: 3px 6.5px 2px 6.5px;
-      bottom: 5px;
-    }
-
-    @include tablet {
-      padding: 5px 6.5px 4px 6.5px;
-    }
+    a {
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:426]: This should not be under the `p, a` above. That should have closed already.

File: assets/sass/cds/_sections.scss [hunk 9]
```diff
@@ -739,95 +769,32 @@ span.livefr {
...
     height: auto;
   }
-
 }
 
 .work-with-us--process {
   border-top: $medium-grey 1px solid;
 }
 
-.work-with-us--apply-now {
```
Comment (dj2) [inline assets/sass/cds/_sections.scss:770]: Where did all of this go?

**Review by dj2 — COMMENTED** [review]
Can we remove the new styles from this PR so it's just formatting changes?  There appears to be something wrong around the `p, a` block I flagged in the review.

File: assets/sass/cds/_sections.scss (hunk 9 above)
Comment (omartehsin1) [reply assets/sass/cds/_sections.scss:770]: Line 1435 _section.scss

**Pull request closed without merging**